tests/test_datasets.py

```diff
@@ -14,7 +14,7 @@
 class TestDataset(object):
     def setUp(self):
         self.data = [1, 2, 3]
-        self.stream = DataStream(IterableDataset(self.data))
+        self.stream = IterableDataset(self.data).get_example_stream()
```

Comment:
Good question. The only place I can think of where we can pass no iteration scheme to `DataStream` is for `IterableDataset` datasets. This doesn't make a very predictable interface.